__tests__/styles.js

```diff
@@ -17,6 +17,44 @@ describe('generator-one-base:styles', () => {
         'src/styles/vendor'
       ]);
     });
+  });
 
+  describe('Sass MQ option', () => {
```

Comment:
yeah, ideally. It adds a lot of overhead to each of the tests, unfortunately. I used `skipInstall: false` in the scripts generator and that added between 15-30 sec to the time of the tests.

Reply:
I'm not super concerned with testing things related to 3rd-parties. It'd be nice to be able to test that the `install` sub-task finished successfully, but double-checking that it installed every single package doesn't seem critical. Otherwise, testing the `gulp` generator `install` process would be ridiculous.

Reply:
Yeah, I'm not talking about making sure they installed, more like making sure we're generating a good `package.json` file (with the deps the user selected). However, I now remember that I tried that and, couldn't because I didn't have a good way to get the version number of the packages, which is why we have to use `yeoman` to install the packages rather than just using it to write the `package.json` file.